### PR TITLE
Release notes for Percona Server 5.6.35-80.1 (5.6)

### PR DESCRIPTION
**Pull request by hrvojem**

### CODE REVIEW

File: doc/source/release-notes/Percona-Server-5.6.35-80.1.rst [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+ Log tracking initialization did not find last valid bitmap data correctly,
+ potentially resulting in needless redo log retracking or hole in the tracked
+ LSN range. Bug fixed :bug:`1658055`.
+
+ If :ref:`audit_log_plugin` was unable to create file pointed by
+ :variable:`audit_log_file`, server would crash during the startup. Bug fixed
+ :bug:`1666496`.
+
+ ``CREATE TEMPORARY TABLE ... SELECT ...`` would get logged in the binary
```
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-5.6.35-80.1.rst]: A "DROP TEMPROARY TABLE ... " for a table created by a "CREATE .... " ....

File: doc/source/release-notes/Percona-Server-5.6.35-80.1.rst [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+ good. Bug fixed :bug:`1671152`.
+
+ Row-based replication events were not reflected in ``Rows_updated`` fields in
+ the :ref:`user_stats` ``INFORMATION_SCHEMA`` tables. Bug fixed :bug:`995624`.
+
+ Server could become unresponsive for some time during the ``Query end`` stage
+ when loading big files using the ``LOAD DATA INFILE``. Bug fixed
+ :bug:`1646100`.
+
+ Compression dictionaries were not being used in partitioned tables. Bug fixed
```
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-5.6.35-80.1.rst]: It was impossible to use column compression dictionaries with partitioned InnoDB tables

File: doc/source/release-notes/Percona-Server-5.6.35-80.1.rst [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+ :bug:`1653104`.
+
+ Error message for SSL cipher has been improved. Bug fixed :bug:`1660339`
```
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-5.6.35-80.1.rst]: Diagnostics for OpenSSL errors have been improved

File: doc/source/release-notes/Percona-Server-5.6.35-80.1.rst [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+ (upstream :mysqlbug:`75311`).
+
+ When ``DuplicateWeedout`` strategy was used for joins use of temporary table
```
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-5.6.35-80.1.rst]: joins, use
was not reported in the query plan info output extension for the slow query log

**Review by laurynas-biveinis — CHANGES_REQUESTED** [review]

File: doc/source/release-notes/Percona-Server-5.6.35-80.1.rst [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+ TABLE ... SELECT ...`` would get logged in the binary log on a disconnect
+ with mixed mode replication. Bug fixed :bug:`1671013`.
+
+ |TokuDB| did not use index with partitioned tables even if cardinality was
+ good. Bug fixed :bug:`1671152`.
+
+ Row-based replication events were not reflected in ``Rows_updated`` fields in
+ the :ref:`user_stats` ``INFORMATION_SCHEMA`` tables. Bug fixed :bug:`995624`.
+
+ Server could become unresponsive for some time during the ``Query end`` stage
```
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-5.6.35-80.1.rst]: A long-running binary log commit would block SHOW STATUS, which in turn could block a number of of other operations such as client connects and disconnects

**Review by laurynas-biveinis — CHANGES_REQUESTED** [review]

**Comment by george-lorch** [comment]
TokuDB and MyRocks (mysqldump --order--by-primary-desc) LGTM

**Review by laurynas-biveinis — APPROVED** [review]

**Pull request merged**